csharp/Klassen/Readme.md

```diff
@@ -44,3 +49,146 @@ public class Freund
+
+Eine Methode ohne Rückgabeparameter ist folgend deklariert. Ein Sichtbarkeitswort folgt das Schlüsselwort ```void```. Innerhalb die geschwomennen Klammern, schreib man dann das, was die Methode machen sollte.
+
+```csharp
+public class Freund 
+{
+    public void SagHallo()
+    {
+        Console.WriteLine("Hallo");
+    }
+}
+```
+
+Eine Methode kann, wie auch Konstruktor, mehrere Eingabeparametern haben.
+
+Eine Methode mit Rückgabeparameter muss anstatt das Wort ```void``` ein Rückgabetyp deklarieren. Methode mit Rückgabeparameter muss immer etwas zurückgeben und zwar mit dem Schlüsserwort ```return```.
+
+```csharp
+public class Freund 
+{
+    public int SagMirDeinAlter()
+    {
+        return 25;
+    }
+}
+```
+
+Wir habe hier einige Klassenkonzepte erläutert. Jetz sehen wir uns an wie man eine Klasse verwenden kann.
+
+Wir deklarieren folgende Klasse ```Freund```:
+
+```csharp
+public class Freund 
+{
+    public Freund(string name)
+    {
+        Name = name;
+    }
+    public string Name { get; set; }
+
+    public int Alter { get; set; }
+
+    public void SagHallo()
+    {
+        Console.WriteLine("Hallo mein Name ist " + Name);
+    }
+}
+```
+Um diese Klasse zu verwende, müssen wir diese Klasse instatieren. Das machen wir mit den _Operator_ ```new```.
+
+```csharp
+public class Program
+{
+    public class Freund 
+    {
+        public string Name { get; set; }
+
+        public int Alter { get; set; }
+
+        public void SagHallo()
+        {
+            Console.WriteLine("Hallo mein Name ist " + Name);
+        }
+    }
+
+    static void Main(string[] args)
+    {
+        Freund neueFreund = new Freund();
+    }
+}
+```
+
+Ab jetzt können wir mit die Variable ```neueFreund``` arbeiten. Zum Beispiel ein _Name_ und ein _Alter_ zu den ```neueFreund``` zuweisen.
+
+```csharp
+class Program
+{
+    public class Freund 
+    {
+        public Freund(string name)
+        {
+            Name = name;
+        }
+        public string Name { get; set; }
+
+        public int Alter { get; set; }
+
+        public void SagHallo()
+        {
+            Console.WriteLine("Hallo mein Name ist " + Name);
+        }
+    }
+
+    static void Main(string[] args)
+    {
+        Freund neueFreund = new Freund("Peter");
+        neueFreund.Alter = 8;
+    }
+}
+```
+
+Und jetzt lassen wir den ```neueFreund``` sich vorstellen und wir sagen ihm ebensfalls Hallo.
+
+```csharp
+class Program
+{
+    public class Freund 
+    {
+        public Freund(string name)
+        {
+            Name = name;
+        }
+        public string Name { get; set; }
+
+        public int Alter { get; set; }
+
+        public void SagHallo()
+        {
+            Console.WriteLine("Hallo mein Name ist " + Name);
+        }
+    }
+
+    static void Main(string[] args)
+    {
+        Freund neueFreund = new Freund("Peter");
+        neueFreund.Alter = 8;
+
+        neueFreund.SagHallo();
+        Console.WriteLine("Hallo " + neueFreund.Name);

```

Comment:
Console.ReadKey(); is missing.

Reply:
Yeah this is generally a problem. When you run it out of the Visual Studio on debug mode, it'll stop on his own. This is only the issue when your run it not in debug mode I think. We leave as is, wen can than discuss it with the children.